techniques/general/G183.html

```diff
@@ -28,11 +32,12 @@ <h3>Colors that would provide 3:1 contrast with black words and 4.5:1 contrast w
          <ol>
             <li>Locate all instances where color alone is used to convey information about text.</li>
             <li>Check that the <a>relative luminance</a> of the color of the text differs from the relative luminance of the surrounding text by a contrast ratio of at least 3:1.</li>
+            <li>Check that pointing (mouseover) to the link causes a visual enhancement (such as an underline, font change, etc.)</li>
```

Comment:
"pointing (mouseover) to the link" ... why not hovering? don't think anybody does anything explicitly on `mouseover` from a JS perspective, it's usually `:hover` from CSS